src/Avalonia.Controls/WindowBase.cs

```diff
@@ -126,6 +126,16 @@ public void Activate()
             PlatformImpl?.Activate();
         }
 
+        public void EnableCloseButton()
+        {
+            PlatformImpl?.EnableCloseButton();
+        }
+
+        public void DisableCloseButton()
```

Comment:
Any reason you implemented these as methods and not as a property? Other similar features are generally implemented as a property.